common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -338,6 +359,19 @@ div.problem {
       }
     }
 
+    &.partially-correct, &.ui-icon-check {
+      p.status {
+        display: inline-block;
+        width: 25px;
+        height: 20px;
+        background: url('../images/partially-correct-icon.png') center center no-repeat;
```

Comment:
I believe this is legacy stuff; there's a line for the correct-icon above that we also aren't using. I may need to update the icon itself to match, though.